src/schema/v2/me/delete_account_mutation.ts

```diff
@@ -0,0 +1,80 @@
+import { GraphQLBoolean, GraphQLNonNull, GraphQLObjectType, GraphQLString, GraphQLUnionType } from "graphql";
+import { mutationWithClientMutationId } from "graphql-relay";
+import { formatGravityError, GravityMutationErrorType } from "lib/gravityErrorHandler";
+import { ResolverContext } from "types/graphql";
+
+
+const AccountMutationDeleteSuccessType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "AccountMutationSuccess",
+  isTypeOf: (data) => {
+    return data._type !== "GravityMutationError"
+  },
+  fields: () => ({
+    success: {
+      type: GraphQLBoolean,
+      resolve: () => true,
+    },
+  })
+})
+
+const AccountMutationFailureType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "AccountMutationFailure",
+  isTypeOf: (data) => {
+    return data._type === "GravityMutationError"
+  },
+  fields: () => ({
+    mutationError: {
+      type: GravityMutationErrorType,
+      resolve: (err) => (typeof err.message === "object" ? err.message : err),
+    },
+  }),
+})
+
+export const AccountMutationType = new GraphQLUnionType({
+  name: "AccountMutationType",
+  types: [AccountMutationDeleteSuccessType, AccountMutationFailureType],
+})
+
+export const deleteUserAccountMutation = mutationWithClientMutationId<
+  any,
+  any,
+  ResolverContext
+>({
+  name: "DeleteAccount",
+  description: "Delete User Artsy Account",
+  inputFields: {
+    id: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
```

Comment:
I know we accept an `explanation` which we'll want to also support here.

Reply:
`url` as well

https://github.com/artsy/gravity/blob/d53fd1a51711b98103feabdc64503be9d893b585/app/api/v1/me_endpoint.rb#L100-L101

Reply:
And actually: we don't want to accept an ID at all. This will be hitting the `me` endpoint; you can only delete the logged in user.

Reply:
Updated.